Test

In [1]:
!python -V

Python 3.9.17


In [2]:
import pandas as pd

You need:

    Python 3.9 (note that videos use 3.8)
    NumPy, Pandas and Scikit-Learn (latest available versions)
    Matplotlib and Seaborn
    Jupyter notebooks

pip install numpy pandas scikit-learn seaborn jupyter xgboost

Later in the course you will also need to install Tensorflow, but we can skip this part for now.

## Reproduce environment